cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1362,7 +1362,7 @@ declare namespace Deno {
     /**
      * An AbortSignal that allows closing the process using the corresponding
      * AbortController by sending the process a SIGTERM signal.
-     * Not Supported by execSync.
+     * Not supported in execSync or spawnSync.
```

Comment:
Actually, `execSync` doesnt exist anymore (its the old name for `spawnSync`), so the text should be updated accordingly